builders/interact/ethereum-api/precompiles/batch.md

```diff
@@ -0,0 +1,286 @@
+
+```text
+[
+  "0x6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b",
+  "0xd14b70a55F6cBAc06d4FA49b99be0370D0e1BD39", 
+  "0xd14b70a55F6cBAc06d4FA49b99be0370D0e1BD39"
+]
+```
+
+There will also be three values for the `value` array. The first address in the `to` input array indicates `1000000000000000000` wei or `1` UNIT of the native token. Remember that the native tokens of Tanssi EVM ContainerChains have [18 decimal points just like Ethereum](https://eth-converter.com/){target=\_blank}. The following two values are `0` because the function that their subtransactions are interacting with does not accept or require native currency.  
+
+```text
+["1000000000000000000", "0", "0"]
+```
+
+You will need three values for the `callData` array. Since transferring native currency does not require call data, the string is simply blank. The second and third values in the array correspond to invocations of **setMessage** that set messages to IDs 5 and 6.
+
+```text
+[
+  "0x", 
+  "0x648345c8000000000000000000000000000000000000000000000000000000000000000500000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000009796f752061726520610000000000000000000000000000000000000000000000", 
+  "0x648345c800000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000e61206d6f6f6e6265616d2070726f000000000000000000000000000000000000"
+]
+```
+
+The final input is for `gas_input`. This array will be left empty to forward all remaining gas to each subtransaction.
+
+```text
+[]
+```
+
+Try sending a batched transaction with these inputs in Remix the same way [you batched a function call](#function-interaction-via-precompile).
+
+And that's it! You've successfully interacted with the ERC-20 precompile using MetaMask and Remix!
+
+## Ethereum Development Libraries {: #ethereum-development-libraries }
+
+If you have followed the [Ethers.js tutorial](/builders/interact/ethereum-api/libraries/ethersjs/){target=\_blank}, you may find it difficult to find the call data for a function. The answer is hidden within Ether's `Interface` object, where the [encodeFunctionData](https://docs.ethers.org/v6/api/abi/#Interface-encodeFunctionData){target=\_blank} function allows you to input your function name and inputs to receive the resultant call data. [Web3.js](/builders/interact/ethereum-api/libraries/web3js/){target=\_blank} has a similar function, [encodeFunctionCall](https://web3js.readthedocs.io/en/v1.2.11/web3-eth-abi.html#encodefunctioncall){target=\_blank}.
```

Comment:
the web3 link is copy/pasta from moonbeam docs and needs to be updated, please double-check links